src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1167,4 +1167,13 @@ Win + Shift + O to toggle your video</value>
   <data name="Run_Radio_Position_Primary_Monitor.Content" xml:space="preserve">
     <value>Primary monitor</value>
   </data>
-</root>
+  <data name="FancyZones_FlashZones.Content" xml:space="preserve">
+    <value>Flash zones when switching layout</value>
+  </data>
+  <data name="FancyZones_QuickLayoutSwitch.Content" xml:space="preserve">
+    <value>Enable quick layout switch</value>
+  </data>
+  <data name="FancyZones_QuickLayoutSwitch_GroupSettings.Text" xml:space="preserve">
+    <value>Quick layout switch</value>

```

Comment:
```suggestion
    <value>Layout features</value>
```

Suggest the text change for better flexibility on new setting in the feature.

Reply:
@htcfreek 
right now that section is specific for the quick layout switch, we should keep the description as it is.
If in the future the section will be expanded, it will be renamed.